src/test/scala/uk/gov/hmrc/api/specs/AuthorisationsSpec.scala

```diff
@@ -253,12 +246,13 @@ class AuthorisationsSpec extends BaseSpec with AuthHelper {
   }
 
   Feature("500, INTERNAL_SERVER_ERROR case Scenarios") {
-    Scenario("500, internal server error") {
+    Scenario("500, wrong content type") {
```

Comment:
Wrong content type shouldn't trigger a 500. I understand the 500 is due to no matching stub, but the wrong content type is misleading. It can just say 500 internal server error from upstream.

You should also add in another 406 test. There are two times you get that error. One if content type header is missing, and one if accept header is missing.